packages/graphql-key-transformer/src/KeyTransformer.ts

```diff
@@ -411,22 +411,47 @@ export class KeyTransformer extends Transformer {
     if (this.isPrimaryKey(directive)) {
       const directiveArgs: KeyArguments = getDirectiveArguments(directive);
 
+      // check @model mutation argument to update schema or not
+      let shouldMakeCreate = true;
+      let shouldMakeUpdate = true;
+      let shouldMakeDelete = true;
+
+      if (ctx.featureFlags.getBoolean('skipOverrideMutationInputTypes', true)) {
+        const modelDirective = definition.directives.reduce(dir => {
```

Comment:
seems like you're trying to filter here, not reduce?

Reply:
yeah filter can also be used here but then I have to use ```modeldirective[0]```. I can make the change.
I went for reduce as it will return an object

Reply:
in that case, I would use .find()

Reply:
done!